admin/class-perfecty-push-admin.php

```diff
@@ -132,8 +132,8 @@ public function register_admin_menu() {
 
 		add_submenu_page(
 			'perfecty-push',
-			esc_html__( 'Notifications', 'perfecty-push-notifications' ),
-			esc_html__( 'Notifications', 'perfecty-push-notifications' ),
+			esc_html__( 'Notifications log', 'perfecty-push-notifications' ),
```

Comment:
The problem with adding `log` is that this section allows you to execute actions on the notifications, which is not what the logs stand for. You usually use logs to see what happened but no actions can be triggered from them directly. We could have call this section `Notifications History`, but again, we allow the users to perform actions like cancel or delete, and probably in the future we will add more advanced operations.

Reply:
What about `feed`?

Reply:
Ok, now I see your point. Then we can call them `Notification jobs` as they are basically that, they hold the information of the different jobs that are processed, and they can be canceled, retried, etc. As you pointed out, they are different to the actual Notifications shown in the statistics. Good catch!